source/_integrations/fireservicerota.markdown

```diff
@@ -0,0 +1,286 @@
+
+| Attribute      | Description                                                                                                                                                                                                                                |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `id` | Incident response id.                                             |
+| `incident_id` | Unique id for active incident.                                             |
+| `membership_id` | Membership id.                                             |
+| `group_id` | Group id.                                             |
+| `user_name` | Your username.                                             |
+| `user_id` | User id.                                             |
+| `assigned_skill_ids` | Assigned skill id's.                                             |
+| `responded_at` | Time responded.                                             |
+| `start_time` | Incident repsonse start time.                                             |
+| `channel` | Channel.                                             |
+| `status` | Status of repsonse, e.g. 'pending'.                                             |
+| `reported_status` | Reported status, e.g. 'shown_up'.                                             |
+| `arrived_at_station` | 'true' or 'false'.                                             |
+| `available_at_incident_creation` | 'true' or 'false'.                                             |
+| `active_duty_function_ids` | Active function id's, e.g. '540'.                                             |
+
+
+## Advanced Configuration
+
+Practical use can be sounding an alarm and/or blinking lights when an emergency incident is received, use text to speech to play incident details while getting dressed, respond with an acknowledge using a door-sensor leaving house or a button or something else connected to Home Assistant to let your teammates know you are underway.
+
+### FireServiceRota Dashboard
+
+If you want to have your browser to function as audio player, and have screen control you need to install the 'browser_mod' integration (via HACS)
+
+Set the device alias below to match your media player id.
+
+You also need to install the following plugins for Lovelace (again HACS is your friend)
+- 'custom:layout-card'
+- 'custom:bignumber-card'
+- 'custom:vertical-stack-in-card'
+- 'custom:home-feed-card'
+
+{% raw %}
+
+```yaml
+browser_mod:
+  devices:
+    3184e47f_bac9b576:  # <--- change this id to your browser id
+      name: bwr-nesthub
+
+sensor:
+  - platform: template
+    sensors:
+      incidents_speech:
+        friendly_name: "Speech Sensor"
+        value_template: "{{ state_attr('sensor.incidents', 'message_to_speech_url') }}"
```

Comment:
I'm confused, why create a template sensor for this?

Reply:
You are right, its too complex/redundant, can I keep the example config basics after a rewrite and I left out the 3rd party stuff?

Reply:
Rewrote all with new 0.115 syntax whenever possible. (couldn't find a string compare option under conditions like for temp above/below)